### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by andreasbossard**
Thank you for this great software! I hope you will find this updates to the readme useful.

I had some problems with the installation instructions:
- what to replace [type] with
- fallbackresource was not supported by my hoster, so I had to use mod_rewrite
- The folder-permissions did not seem to be 755, so I had to change this in order for packages.json to show up correctly and not show a 404 error.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -39,20 +39,34 @@ It will serve the following Composer repository at `/packages.json` automagicall
...
-Create the project:
+### 1. Create the project:
 
 ```
 composer create-project rarst/release-belt
 ```
 
-Place release ZIPs into `/releases/[type]/[vendor]/`.
+### 2. Place release ZIPs into `/releases/[type]/[vendor]/`. 
+`[type]` could be e.g. "symfony-bundle", "wordpress-plugin or "typo3-cms-extension".
```
Comment (Rarst) [inline README.md]: Technically the explicitly supported types right now are `library`, `wordpress-plugin`, and `wordpress-theme`, could you please use these for examples?

Arbitrary type would work, but will not do any additional tweaking (such as requiring `composer/installers`).

File: README.md [hunk 2]
```diff
@@ -39,20 +39,34 @@ It will serve the following Composer repository at `/packages.json` automagicall
...
+For example with the following `.htaccess`:
 
 ```
 FallbackResource /index.php
 ```
 
+If your server does not support FallbackResource, you can use mod_rewrite in your `.htaccess` with this code:
+
+```
+RewriteEngine On
```
Comment (Rarst) [inline README.md]: Please wrap these directives in a check if `mod_rewrite` is available, or they would crash the site for someone trying them without it compiled into Apache.

File: README.md [hunk 3]
```diff
@@ -39,20 +39,34 @@ It will serve the following Composer repository at `/packages.json` automagicall
...
+RewriteCond %{REQUEST_FILENAME} !-f
+RewriteCond %{REQUEST_FILENAME} !-d
+RewriteRule . /index.php [L]
+```
+Make sure, that the folder, which contains the `index.php` has FTP-Permissions `755`
```
Comment (Rarst) [inline README.md]: Permissions are way too specific to hosting, what fixes it on one host breaks it on another. Please drop this line.

File: README.md [hunk 4]
```diff
@@ -39,20 +39,34 @@ It will serve the following Composer repository at `/packages.json` automagicall
...
+
+### 4. Test if `index.php` and `packages.json` is shown
```
Comment (Rarst) [inline README.md]: 1. This heading should be below PHP server configuration example, which belongs in previous section.
2. I don't like wording, as in what exactly "test" implies. I would suggest something simpler like "Visit the site in browser to check it is working".

**Review by Rarst — CHANGES_REQUESTED** [review]
Cheers! I've requested some tweaks, if you have time. :)

File: README.md (hunk 3 above)
Comment (andreasbossard) [reply README.md]: The problem I had is, that packages.json gave me a 404-error, even though I did the installation correctly. 

Could we then add a troubleshooting-section, or can we add it to the FAQ? "What do I do, if packages.json shows me a 404 error?"

Comment (andreasbossard) [reply README.md]: I added a troubleshooting section. What do you think?

**Comment by andreasbossard** [comment]
@Rarst Thank you for your input. Made the changes you requested. Please review.

File: README.md (hunk 3 above)
Comment (Rarst) [reply README.md]: In my opinion it's not up to _this_ project to teach anyone to configure a web server really. There are a lot of ways to run PHP out there, and as above they are full of nuances from mutually incompatible to creating security holes.

I would still prefer you just drop mention of folder permissions.

For the troubleshooting purposes I would just say "If you have further trouble with configuration please refer to your hosting documentation or support for recommended settings".

Comment (andreasbossard) [reply README.md]: I removed the troubleshooting section. I think it confuses more than it helps.

**Review by Rarst — APPROVED** [review]

**Comment by Rarst** [comment]
Cheers, merged. Feel free to let me know how the project works out for your needs. :)

**Pull request merged**